src/main.cpp

```diff
@@ -3906,7 +3906,7 @@ void game_loop(void)
       {
         if (game.numfield_15 == -1)
         {
-          set_player_instance(player, PI_HeartZoom, 0);
+          //set_player_instance(player, PI_HeartZoom, 0);
```

Comment:
perhaps this should be a setting so it can be left on when switching branchestc and without ever accidentally including it in a commit